model/policy.go

```diff
@@ -128,28 +144,33 @@ func (model Model) RemovePolicy(sec string, ptype string, rule []string) bool {
 }
 
 // RemovePolicies removes policy rules from the model.
-func (model Model) RemovePolicies(sec string, ptype string, rules [][]string) bool {
-	OUTER : for j := 0 ; j < len(rules) ; j++ {
+func (model Model) RemovePolicies(sec string, ptype string, rules [][]string) (bool, [][]string) {
+OUTER:
+	for j := 0; j < len(rules); j++ {
 		for _, r := range model[sec][ptype].Policy {
 			if util.ArrayEquals(rules[j], r) {
 				continue OUTER
 			}
 		}
-		return false
+		return false, nil
```

Comment:
same here

Reply:
Returns `nil` will convert zero slice, it's working.